libs/klv/src/main/java/org/codice/alliance/libs/klv/AttributeNameConstants.java

```diff
@@ -57,11 +65,11 @@
 
     public static final String OBJECT_COUNTRY_CODES = "object-country-codes";
 
-    public static final String CHECKSUM = "checksum";
+    public static final String CHECKSUM = "klv-packet-checksum";
 
     public static final String IMAGE_COORDINATE_SYSTEM = "image-coordinate-system";
```

Comment:
The KLV docs say: String of the image coordinate 
system used. 
e.g.: 'Geodetic WGS84', 
'Geocentric WGS84', 'UTM', 
'None', etc. 
Maximum 127 characters.